src/app/src/App.css

```diff
@@ -719,4 +719,32 @@ hr {
 .cell-tooltip {
   font-size: 12px !important;
   margin: 0 !important;
+}
+
+.facility-list-row__expanded td {
+  border: none;
+}
+
+.facility-list-row__expanded + .facility-list-row__expanded td {
+  border-bottom: 1px solid #e0e0e0
+}
+
+.STATUS_MATCHED_COLLAPSED {
```

Comment:
Consider updating the comment in api/models.py and explain that if a new status is added we should update this stylesheet as well https://github.com/open-apparel-registry/open-apparel-registry/blob/783e782349255d1ea10241c56983c2dd07f753b0/src/django/api/models.py#L489-L490